sections/background.md

```diff
@@ -14,4 +14,10 @@ In the next years, the MATF ensured that mobile considerations were included in
 - 2.5.8 Target Size (Minimum) (AA), to providing minimum touch target sizes;
 - 3.3.7 Redundant Entry (A), to reduce the amount of repetitive typing.
 
-In January 2024, the MATF regrouped and welcomed new participants to work on updated guidance for applying WCAG 2.2 to Mobile Applications. Since then, the group has worked on creating this updated document, with the intention of publishing it as a Group Note.
+In January 2024, the MATF regrouped and welcomed new participants to work on updated guidance for applying WCAG 2.2 to Mobile Applications.
+
+This current document, “Guidance on Applying WCAG 2.2 to Mobile (WCAG2Mobile)” is specific to how WCAG applies to Mobile Applications, including but not limited to native mobile apps, mobile web apps, mobile web content, and hybrid apps using web components inside native mobile apps.
```

Comment:
"not limited to", I am wondering if we are not a little bit too open here. We have had some discussions about the scope of this document, for example if smartwatches apps would be part of these guidelines or not. When reading this sentence, I have the feeling that it would also be part of it.

Reply:
Thx @AlainVagner for mentioning this, will achieve this in the scope.